ykman/cli/__main__.py

```diff
@@ -201,6 +202,18 @@ def list_keys(ctx, serials, readers):
     """
     List connected YubiKeys.
     """
+
+    def print_device(dev, serial):
```

Comment:
Maybe call it `_print_device` ?

Reply:
Sure, but is that necessary for a nested function?

Reply:
Done.